src/ui/components/shared/SettingsModal/ReplayInvitations.tsx

```diff
@@ -27,7 +28,7 @@ export default function ReplayInvitations() {
     );
   }
 
-  if (!invited) {
+  if (availableInvitations === null) {
```

Comment:
How do you feel about this check? I admit I'm tempted to add `internal` and `invited` booleans on `viewer` so if you haven't been invited, it's just `availableInvitations === 0`

Reply:
> I admit I'm tempted to add internal and invited booleans on viewer

Yes please

Reply:
I forgot to ping you on the issue, but I landed this last night https://github.com/RecordReplay/backend/pull/1827

Reply:
That change also means this is never going to pass right now, since doesn't return `null` anymore.